### ISSUE TITLE
Data Structure Changed in firmware v3.88 node crashing

### ISSUE THREAD

**Issue by BGCDE** [Bouni/node-red-contrib-luxtronik2-ws#11]
I've fixed luxtronik2-ws.js preventing the crash and adding all new attibutes. 
I can create a pull request. Although i am a github newbie.

**Comment by lessthanmore1** [comment]
I did the update before checking here so, I run into the same issue.
Would be nice if it could be fixed.
@BGCDE I use node-red as an adapter within ioBroker; do you know where to find the *.js file so I can manually fix it?

Thank you in advance

**Comment by Bouni** [comment]
Can you somehow evaluate that this also works with heatpumps < V3.88?
I want to avoild make it working for you but break it for all other users ...

**Comment by BGCDE** [comment]
No, firmware downgrade is not possible. I asked in htdialog firmware thread, let's wait for feedback. On the other side ait did not change any basics, just adding a 2nd level for 2 item groups as you named them. All other groups have the old format with no change.

**Comment by Henriettel** [comment]
Any news on this? Updated my WP today to V3.88 and now i receive no values anymore -.-
If you need help or a tester, let me know

**Comment by BGCDE** [comment]
> Any news on this? Updated my WP today to V3.88 and now i receive no values anymore -.- If you need help or a tester, let me know

No feedback until now. You can manually copy the js file from my fork to confirm.

**Comment by Henriettel** [comment]
I tested the file from the PR. Works fine for 5 days now.
